src/helpers/checkAttestationHash.ts

```diff
@@ -1,5 +1,15 @@
 import ketlAttestationContract from 'helpers/ketlAttestationContract'
 
-export default function checkAttestationHash(hash: string) {
-  return ketlAttestationContract.attestationHashesEntangled(hash)
+export default async function checkAttestationHash(
+  hash: string,
+  attestationType: number
```

Comment:
Should be `enum`

Reply:
Agreed, but would prefer to do this in a separate PR.